containers/kubernetes/install.sh

```diff
@@ -11,13 +11,13 @@ if [ -z "$OFFLINE" ]; then
 fi
 
 # Install packages without dependencies
-/container/scripts/install-rpms.sh cockpit-ws cockpit-dashboard
+/container/scripts/install-rpms.sh cockpit-ws
 /container/scripts/install-rpms.sh --nodeps cockpit-bridge
```

Comment:
I think we need to install libssh in this container too for some corner use cases. I think --nodeps may interfere with that.

Reply:
This would again be missing libpolkit (although I haven't checked in this container, perhaps it's already getting pulled in by something else). But I agree that we should explicitly install libssh here.

Reply:
Note for next week: We can't call dnf during the test suite, but we can put it into here:
```
if [ -z "$OFFLINE" ]; then
    "$INSTALLER" -y update
fi
```
During tests, the cockpit/base container already installs libssh explicitly; and if that would be wrong, we'd just get test failures. This would purely be for making sure that the released container has it. I'll push that after we resolve the -ws vs. -bridge question.